content/_data/changelogs/lts.yml

```diff
@@ -140,6 +140,64 @@
       message: Do not fail to write a log file just because something deleted the parent directory.
       issue: 16634
       pull: 2738
+- version: "2.46.1"
+  date: 2017-03-19
+  lts_predecessor: "2.32.3"
+  lts_baseline: "2.46"
+  lts_changes:
+    - type: bug
+      message: >
+        Prevent file descriptor leaks when Windows Service installer fails to read data from the service startup log.
+      issue: 42670
+      pull: 2793
+    - type: bug
+      message: >
+        Update Remoting from <code>3.5</code> to <code>3.7</code>
+        in order to prevent file descriptor leaks on agents in the case of multiple connection attempts.
+      references:
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#37
+          title: full changelog
+      issue: 42371
+      pull: 2773
+    - type: bug
+      message: Exceptions during Jenkins cleanup step should not block restart.
+      issue: 42164
+    - type: bug
+      message: Cryptic error message when loading JnlpSlaveAgentProtocol4. <<< AUTOGENERATED (https://issues.jenkins-ci.org/browse/JENKINS-41987)
```

Comment:
That is a reminder to review the generated entry and delete the note. It clearly does not work ...